tap_clickup/client.py

```diff
@@ -128,15 +121,21 @@ def from_parent_context(self, context: dict):
         if self.partitions is None:
             return context
         else:
-            # Was going to copy the partitions, but the _sync call, forces us
-            # To use partitions, instead of being able to provide a list of contexts
-            # Ideally we wouldn't mutate partitions here, and we'd just provide
-            # A copy of partitions with context merged so we don't have side effects
-            # Not certain why pylint needs this partitions is iterable
-            # We check the None case above
-            for partition in self.partitions:  # pylint: disable=not-an-iterable
-                partition.update(context.copy())  # Add copy of context to partition
-            return None  # Context now handled at the partition level
+            #Goal here is to combine Parent/Child relationships with Partions
+            #Another way to think about this is that Partitions are now
+            #Lists of contexts, used to create multiple requests based on one context.
```

Comment:
This line is hard to understand. 

Partitions are now "context multipliers", used to create multiple context's, which ultimately creates `len(self.base_partition)` * `context` number of requests. 

Still not a great definition, needs work :/